wear/src/main/java/io/homeassistant/companion/android/home/MainViewModel.kt

```diff
@@ -224,7 +233,7 @@ class MainViewModel @Inject constructor(
     }
 
     suspend fun entityRegistryUpdates() {
-        if (!homePresenter.isConnected())
+        if (!homePresenter.isConnected() || isFavoritesOnly)
```

Comment:
Why not do updates even though you do load the entity registry initially? (l191)

Reply:
I was only interested in the initial list of entities, but reading this comment. Yea we should make sure the list is up to date if modified when the app is open :)